cylc/flow/prerequisite.py

```diff
@@ -236,7 +237,7 @@ def api_dump(self, workflow_id):
         num_length = math.ceil(len(self.satisfied) / 10)
         for ind, message_tuple in enumerate(sorted(self.satisfied)):
             name, point = message_tuple[0:2]
-            t_id = f"{workflow_id}/{point}/{name}"
+            t_id = f"{workflow_id}{ID_DELIM}{point}{ID_DELIM}{name}"
```

Comment:
Doesn't matter that this was a different delimiter here? (`/` vs `|`)

Reply:
(Or was that the bug?  it's used in `PbConditions`)

Reply:
Yes, this was the bug that needed fixed. But can't import from `cylc.flow.data_store_mgr`.